scripts/dir.js

```diff
@@ -0,0 +1,90 @@
+//  A script that returns the directory url of the user
+//
+// Commands
+//   webbot dir <search_query>: returns directory url of the user
+//
+// URLS:
+//   POST /hubot/dir
+//     Follows format suggested here: https://docs.mattermost.com/developer/slash-commands.html
+//     data:
+//       token: Should be similar than MATTERMOST_TOKEN_CMD_DIR
+//       text: query search for
+//     response:
+//       {"response_type": "ephemeral", "text": TEXT_POSTED}
+//
+// Authors
+//   Hackathon
+
+const MATTERMOST_TOKEN_CMD_DIR = process.env.MATTERMOST_TOKEN_CMD_DIR;
+if (!MATTERMOST_TOKEN_CMD_DIR) {
+  console.log("Missing MATTERMOST_TOKEN_CMD_DIR in environment");
+}
+
+async function generateDirectoryLink(argument) {
```

Comment:
This function doesn't need to be async since it's not awaiting any promises or other async functions. You can remove all the `async`s and `await`s from this file and it will still work.